contrib/kube-prometheus/jsonnet/kube-prometheus/kube-prometheus.libsonnet

```diff
@@ -72,6 +74,7 @@ local configMapList = k.core.v1.configMapList;
 
     prometheus+:: {
       rules: $.prometheusRules + $.prometheusAlerts,
+      enableAdminAPI: $._config.enableAdminAPI,
```

Comment:
jsonnet has merge semantics, therefore we don't need to specify this here at all